src/libs/antares/inifile/include/antares/inifile/inifile.h

```diff
@@ -43,22 +42,21 @@ class IniFile final
     class Property final
     {
     public:
-        Property();
+        Property() = default;
         explicit Property(const AnyString& key);
         template<class U>
         Property(const AnyString& key, const U& value);
-        ~Property();
+        ~Property() = default;
 
-        template<class StreamT>
-        void saveToStream(StreamT& file, uint64_t& written) const;
+        void saveToStream(std::ostream& file, uint64_t& written) const;
```

Comment:
ofstream is more suitable for "file", unless file is not only a file

Reply:
We use `std::stringstream` for unit tests (in-memory tests)